### DISCUSSION TITLE
Tank drone POI

### DISCUSSION THREAD

**Discussion by scarf005** [cataclysmbn/Cataclysm-BN#4378] [General]
## Summary

create a tank drone point of interest, like mi-go tower or fungus tower.

## Why?

Tankbots are obsoleted, but they're cool concept i'd like to see in-game someday.
However, there's reason it got obsoleted; It spawned out of nowhere and instakilled you from distance.

## Solution

1. create a new POI, e.g 'high threat barricade'. could be spawned near military base, surface lab or bridge.
2. make it become hostile only when player approaches. (e.g 10 tiles)
3. rollback its arsenal, e.g cannons, missiles, flamethrowers
4. make it guard high-tier loot such as nanomaterials or power armor.

**Comment by Firestorm01X2** [comment]
I am totally for it. It seems for a lot of players it is game "signature enemy". 
It is better just do not make it one-shot machine for prepared player. Also we have laser pointer aiming fixed.
